src/sentry/tasks/post_process.py

```diff
@@ -120,6 +122,24 @@ def _capture_group_stats(job: PostProcessJob) -> None:
     metrics.incr("events.unique", tags=tags, skip_internal=False)
 
 
+def should_issue_owners_ratelimit(
+    cache_key,
+):
```

Comment:
Could you please a docstring state the purpose?

For instance something like this:
```
Make sure that we do not make more requests than ISSUE_OWNERS_PER_PROJECT_PER_MIN_RATELIMIT at the project level.
```